src/js/edu-benefits/5490/config/form.js

```diff
@@ -312,6 +314,9 @@ const formConfig = {
           },
           schema: {
             type: 'object',
+            // If answered 'Yes' without entering information, it's the same as
+            //  answering 'No' as far as the back end is concerned.
+            required: ['view:applicantServed'],
```

Comment:
It does seem a little weird, but I think the idea is just to force that you read the question and make a choice.